main/signers/trezor/Trezor/index.ts

```diff
@@ -32,11 +32,11 @@ export const Status = {
   ENTERING_PASSPHRASE: 'waiting for input on device'
 }
 
-function createErrorMessage(message: string, cause: string = '') {
+function createError(message: string, code: string, cause: string = '') {
   // the cause may need to be transformed into a more informative message
   return cause.toLowerCase().match(/forbidden key path/)
-    ? 'derivation path failed strict safety checks on trezor device'
-    : message
+    ? new DeviceError('derivation path failed strict safety checks on trezor device', 'SAFETY_CHECKS')
```

Comment:
I'd assign this message to a Status enum value and then update the map below

Reply:
I actually want to get away from that going forward as I think we've too heavily conflated enumerable _states_ (ie "ready" or "disconnected") with arbitrary textual status which should be able to be anything we want to display. that's part of the distinction I want to make once we get a chance to re-work all the signer stuff